### PR TITLE
Release AD7134

### PR DESCRIPTION
**Pull request by jansunil**
Release AD7134 project with the STM32H563 support

### CODE REVIEW

File: projects/ad7134_iio/app/app_config.c [hunk 1]
```diff
@@ -172,7 +170,7 @@ static struct eeprom_24xx32a_init_param eeprom_extra_init_params = {
 
 /* EEPROM init parameters */
 static struct no_os_eeprom_init_param eeprom_init_params = {
-	.device_id = 0,
+	.device_id = I2C_DEVICE_ID,
```
Comment (ribdp) [inline projects/ad7134_iio/app/app_config.c:173]: The macro would also need to be added in app_config_mbed.h I think?

File: projects/ad7134_iio/app/app_config_stm32.c [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * @brief   STM32 Specific configuration files for AD7134 IIO Application
  * @details This module contains the STM32 platform specific configurations
 ********************************************************************************
-* Copyright (c) 2021 Analog Devices, Inc.
+* Copyright (c) 2021,2023 Analog Devices, Inc.
```
Comment (ribdp) [inline projects/ad7134_iio/app/app_config_stm32.c:6]: Should this be 2024?

File: projects/ad7134_iio/app/app_config.c (hunk 1 above)
Comment (jansunil) [reply projects/ad7134_iio/app/app_config.c:173]: It is already present in the mbed config-https://github.com/analogdevicesinc/precision-converters-firmware/blob/release-ad777x-h563/projects/ad7134_iio/app/app_config_mbed.h#L85

Comment (ribdp) [reply projects/ad7134_iio/app/app_config.c:173]: My bad, missed that somehow

**Comment by D-Disha** [comment]
Can the second commit message also include a mention of the tests folder update

File: libraries/no-OS.lib [hunk 3]
```diff
@@ -1 +1 @@
-https://github.com/analogdevicesinc/no-OS/#926e22ab5b1309d62a25873b8b07bf581e674c32
\ No newline at end of file
+hhttps://github.com/analogdevicesinc/no-OS/#68bf422d83850074e3d721e7e3cc6b67e22bc433
```
Comment (SaikiranGudla) [inline libraries/no-OS.lib:1]: Correction: `https`

**Review by SaikiranGudla — APPROVED** [review]

File: projects/ad7134_iio/app/app_config_stm32.c (hunk 2 above)
Comment (jansunil) [reply projects/ad7134_iio/app/app_config_stm32.c]: Updated the copyrights

File: libraries/no-OS.lib (hunk 3 above)
Comment (jansunil) [reply libraries/no-OS.lib]: Updated the link

**Review by ribdp — APPROVED** [review]

**Review by SaikiranGudla — DISMISSED** [review]

**Comment by jansunil** [comment]
changelog:

Updated the libraries/.mbedignore to exclude the new ad796x drivers added in the no-OS repo
Updated the .extsettings file to include the no_os_dma.c file

**Review by SaikiranGudla — APPROVED** [review]

**Review by ribdp — APPROVED** [review]

**Pull request merged**